Stucco/template/psakeFile.ps1

```diff
@@ -3,6 +3,7 @@ properties {
     # This modifies the default behavior from the "Build" task
     # in the PowerShellBuild shared psake task module
     $PSBPreference.Build.CompileModule = $false
+    $PSBPreference.Help.DefaultLocale = 'en-US'
```

Comment:
Without a default value here, the GENERATEMARKDOWN task failed on the Linux build.

Reply:
There is a fix in PowerShellBuild coming for this. It will default to `en-US` if `(Get-UICulture).Name` is null